### ISSUE TITLE
Lenses for configuration

### ISSUE THREAD

**Issue by jfischoff** [jfischoff/tmp-postgres#35]
Having stock lenses for configuration might make `mappend`ing to `defaultConfig` easier.

So 

```haskell
defaultPostgresConf extra = defaultConfig <> mempty
  { configPlan = mempty
    { partialPlanConfig = extra
    }
  }
```

vs. 

```haskell
defaultPostgresConf extra = defaultConfig & configPlanL . partialPlanConfigL <>~ extra
```

**Comment by jfischoff** [comment]
I think I could provide the lenses without depending on any lens package and not use lenses internally but suggest one could in the doc.

**Comment by jfischoff** [comment]
Addressed in 0c9c72d
